Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.20.10 AS builder
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+WORKDIR /qe-tools
+
+COPY . .
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o /tmp/qe-tools main.go
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.3
+
+COPY --from=builder /tmp/qe-tools /usr/bin/
+COPY --from=builder /qe-tools/config /config
+COPY --from=builder /qe-tools/entrypoint.sh /entrypoint.sh
+
+USER 65532:65532
+
+ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
my bad, it's actually not needed in this type of container, sorry. it'll work even without it